coral-hive/src/main/java/com/linkedin/coral/hive/hive2rel/parsetree/ParseTreeBuilder.java

```diff
@@ -196,13 +199,21 @@ private SqlNode visitLateralViewInternal(ASTNode node, ParseContext ctx, boolean
     checkState(sqlNodes.size() == 2 && sqlNodes.get(0) instanceof SqlNodeList);
 
     // rightNode is AS.createCall(unnest, t, col)
-    SqlNode rightNode = ((SqlNodeList) sqlNodes.get(0)).get(0);
+    SqlNode rightNode = Iterables.getOnlyElement((SqlNodeList) sqlNodes.get(0));
     checkState(rightNode instanceof SqlCall);
     SqlCall aliasCall = (SqlCall) rightNode;
     checkState(aliasCall.getOperator() instanceof SqlAsOperator);
     List<SqlNode> aliasOperands = aliasCall.getOperandList();
-    checkState(aliasOperands.size() == 3 && aliasOperands.get(0) instanceof SqlCall);
-    SqlCall unnestCall = (SqlCall) aliasOperands.get(0);
+    checkState(aliasOperands.get(0) instanceof SqlCall);
+    SqlCall sqlCall = (SqlCall) aliasOperands.get(0);
```

Comment:
Rename `sqlCall` to `userDefinedTableFunction`?

Reply:
Before my change it was called `unnestCall`. I thought the name not appropriate as i don't know whether UNNEST is an operation in Hive, but this can be an explode (like unnest) or something else.
I don't find `userDefinedTableFunction` an appropriate name, as i don't know whether this is user-defined.

Reply:
Ok fair point about user-defined. How about `tableFunction`?

Reply:
changed